docs/INSTALLATION.md

```diff
@@ -1,13 +1,15 @@
 ## Dart
-* [Install Dart](https://www.dartlang.org/install)
-* Add the Dart SDK to your path.
-* From a new command window, type `dart --version` to verify that Dart is now available.
+* One option is to follow the [instructions for installation on the Dart website](https://www.dartlang.org/install).
 
-## Dart Plugin for your IDE or Editor
-* If you are using an IDE or an editor, [install and configure the Dart plugin](https://www.dartlang.org/tools).
+### Manual install instruction
+* Download a zip file from [this page](https://www.dartlang.org/install/archive).
+* Unzip the file into whatever directory you choose.
+* Add the full path of that directory to your PATH.
+    * Follow the links below for updating your PATH on Windows, Linux and Mac platforms:
+        * [Instructions for Windows](https://www.computerhope.com/issues/ch000549.htm)
+        * [Instructions for Linux](https://www.computerhope.com/issues/ch001647.htm)
+        * [Instructions for Mac](https://www.pegaxchange.com/tag/macos-sierra-add-entry-to-path-variable/)
+* From a new command window or terminal, type `dart --version` to verify that Dart is now available. If the output shows the version number, then Dart is correctly installed on your computer.
```

Comment:
Idk if it's required or if it'll help. 
Maybe add a tip explaining something like , 'in some cases the updated values for PATH variable don't take effect untill the terminal is restarted/refreshed or a new one is stared. It's better to start a new session.' This will go to the exercism website dart page right?

Reply:
The links provided for all 3 platforms might or might not contain this tip but since we are providing an instruction `dart --version`  for confirmation of PATH variable it should have this tip.

Reply:
That's a valid point, but at least the Linux and macOS instructions already cover this.

Reply:
I though this was covered by stating "From a new command window or terminal ..."

Reply:
It might very well be an extreme case and never happen. This is why I chose to comment instead of requesting changes.

Reply:
Thanks @kabiir for clarifying. That's a good perspective I hadn't considered. I separated the verify step into another section.